README.md

```diff
@@ -1 +1,72 @@
-# bitrise-step-save-npm-cache
\ No newline at end of file
+# Save NPM Cache (Beta)
+
+[![Step changelog](https://shields.io/github/v/release/bitrise-steplib/bitrise-step-save-npm-cache?include_prereleases&label=changelog&color=blueviolet)](https://github.com/bitrise-steplib/bitrise-step-save-npm-cache/releases)
+
+Saves node_modules dependencies. This Step needs to be used in combination with **Restore NPM Cache**.
+
+<details>
+<summary>Description</summary>
+
+Saves node_modules dependencies. This Step needs to be used in combination with **Restore NPM Cache**.
+
+This Step is based on [key-based caching](https://devcenter.bitrise.io/en/builds/caching/key-based-caching.html) and sets up the cache key and path automatically for NPM dependencies. If you'd like to change the cache key (or paths to cache), you might want to use the generic [Save cache](https://github.com/bitrise-steplib/bitrise-step-save-cache) Step instead.
+
+#### Related steps
+
+[Restore NPM cache](https://github.com/bitrise-steplib/bitrise-step-restore-npm-cache/)
+
+[Save cache](https://github.com/bitrise-steplib/bitrise-step-save-cache/)
+
+</details>
+
+## 🧩 Get started
+
+Add this step directly to your workflow in the [Bitrise Workflow Editor](https://devcenter.bitrise.io/steps-and-workflows/steps-and-workflows-index/).
+
+You can also run this step directly with [Bitrise CLI](https://github.com/bitrise-io/bitrise).
+
+⚠️ **Beta status**: While this Step is in beta, everyone can use it without restrictions, quotas or costs.
+
+### Examples
+
+Check out [Workflow Recipes](https://github.com/bitrise-io/workflow-recipes#-key-based-caching-beta) for other platform-specific examples!
```

Comment:
Just noting that we don't have examples here related to the platform specific cache steps (like here https://github.com/bitrise-io/workflow-recipes/blob/main/recipes/rn-key-cache-dependencies.md).